packages/sveltekit/src/client/load.ts

```diff
@@ -270,3 +272,14 @@ function addFetchBreadcrumb(
     },
   );
 }
+
+type MaybeClientWithGetIntegrationsById =
+  | (Client & { getIntegrationById?: BaseClient<ClientOptions>['getIntegrationById'] })
+  | undefined;
+
+type ClientWithGetIntegrationById = Required<MaybeClientWithGetIntegrationsById> &
+  Exclude<MaybeClientWithGetIntegrationsById, undefined>;
+
+function isValidClient(client: MaybeClientWithGetIntegrationsById): client is ClientWithGetIntegrationById {
+  return !!client && typeof client.getIntegrationById === 'function';
```

Comment:
m: Why are we additionally making sure `getIntegrationById` is there? That method is on the BaseClient so I think we can be pretty sure it's there. I think a check if the client exists at all is enough no?

Reply:
For the unlikely case that anyone would use a custom client that doesn't extend BaseClient. I'm aware that this is unlikely but then again, the check doesn't hurt us too much

Reply:
Got it. Sounds good!